Fixed Portuguese and Polish plurals and German translations

VincentTam opened the pull request:
https://github.com/marcelkraus/twietesla/blob/978fe99689adc1c38ce064f474ca554dfbcdfc1f/i18n/de.toml

<!---
  ## Prerequisites
  - I am running the latest version of [Hugo](https://github.com/gohugoio/hugo/releases)
  - I am using the latest version of [Hugo-Future-Imperfect-slim](https://github.com/pacollins/hugo-future-imperfect-slim/releases)
  - I checked the [issues](https://github.com/pacollins/hugo-future-imperfect-slim/issues?utf8=%E2%9C%93&q=is%3Aissue) to make sure that this feature has not been rejected before
  - I checked the [pull requests](https://github.com/pacollins/hugo-future-imperfect-slim/pulls?utf8=%E2%9C%93&q=is%3Apr) to make sure that this feature is not already being developed
-->

<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

Polish and Brazilian's plural was broken.
Updated on 21 Aug 2019: [Polish plural form has been fixed](https://discourse.gohugo.io/t/plural-doesnt-work-for-brazillian-portuguese-and-polish/20329/2?u=vincenttam), and later the Brazilian Portuguese plural has also seen corrected.

I've improved the French translation and provided a sample "German" post (with English contents and front matter "translated" by Google).  The point here is to show that our theme supports German content, so it's not important to find correct text here.  We're open to PR's for improving that file though.

![Screenshot_2019-08-21 Hugo Future Imperfect Slim auf deutsch](https://user-images.githubusercontent.com/5748535/63451543-bed59780-c444-11e9-9e61-93aa86701381.png)

## Motivation and Context
<!---
  Why is this change required? What problem does it solve?
  If it fixes an open issue, please link to the issue here by writing "Closes #XXX"
-->

Fixed #74 and added German translation.

## Checklist:
<!---
  Please review the following points and put an `x` in all the boxes that apply.
  If you're unsure about any of these, don't hesitate to ask. We're here to help!
-->

- [x] I have read the [Contributing Document](https://github.com/pacollins/hugo-future-imperfect-slim/blob/master/.github/CONTRIBUTING.md).
- [x] My code follows the [code style](https://github.com/pacollins/hugo-future-imperfect-slim/blob/master/.github/CONTRIBUTING.md#Style-Guide) of this project.
- [ ] My change requires a change to the [documentation](https://github.com/pacollins/hugo-future-imperfect-slim/wiki).
- [ ] I have updated the documentation, including `theme.toml`, accordingly.

pacollins commented on exampleSite/config.toml:
```diff
@@ -144,6 +144,43 @@ disableLanguages        = [""]
     LanguageName        = "English"
     weight              = 1
 
+  [Languages.de-AT]
```
I'm curious your thoughts on this. So far, we have basically been any language with an i18n to the config file. I'm beginning to wonder if it becomes bloated.

VincentTam replied:
I wanted to switch that back to `de` when the suggestions from my Hugo Forum question failed me.  After this successful trial, which showed the way for displaying both types of :cn: :taiwan: Chinese characters, I'm going to step back and change this back to `de`, unless someone can show the difference between the two, like ["71" in French](https://fr.wikipedia.org/wiki/Nombres_en_fran%C3%A7ais#France,_Canada_et_les_pays_d'Afrique_francophones): *soixante-et-onze* :fr: vs :switzerland: *septante et un*.

I'm doing this for a merely technical reason: I believe that German and Austrian users favor `foo.de.md` over `foo.de-AT.md` or ... , given that both groups of users admit that the difference between the two variants doesn't concern our theme.  Since I've failed to get sth like

```
[Language.de]
  languageCode = "de-at"
```

working, I've drop the country code `at` in minutes.

Note that the above technical reason *doesn't* apply to :cn: zh-CN & :taiwan: zh-TW since :cn: PRC is using a "simplified" (IMO, crippled) version of traditional Chinese characters, so the visual distinction between the two is clear.  To use zh-TW and zh-CN (in parallel), the country code `-tw`/`-cn` is required in every places.  If anyone can point out the difference between :hong_kong: zh-HK and :taiwan: zh-TW (say :bus: :hong_kong: *巴士* vs :taiwan: *公交車*) in our theme, I'll be happy to welcome `i18n/zh-HK.toml` into our theme.

Lemme continue this tmr.

VincentTam commented:
Related: https://github.com/gohugoio/hugo/issues/3846

VincentTam commented:
Reposted with some minor edits for record:

I wanted to switch that back to `de` when I made the first commit .  After the successful trial before the 2nd commit, which showed the way for displaying both types of :cn: :taiwan: Chinese characters, I stepped back and changed this back to `de`.

Unless someone can show the difference between the two, like ["71" in French](https://fr.wikipedia.org/wiki/Nombres_en_fran%C3%A7ais#France,_Canada_et_les_pays_d'Afrique_francophones): *soixante-et-onze* :fr: vs :switzerland: *septante et un*, I'll stick to `de` without country code.

I'm doing this for a merely technical reason: I believe that German and Austrian users favor `foo.de.md` over `foo.de-AT.md` or ... , given that both groups of users admit that the difference between the two variants doesn't concern our theme.  Since I've failed to get sth like

```
[Language.de]
  languageCode = "de-at"
```

working, I've dropped the country code `at` in minutes.

Note that the above technical reason *doesn't* apply to :cn: zh-CN & :taiwan: zh-TW since :cn: PRC is using a "simplified" (IMO, crippled) version of traditional Chinese characters, so the visual distinction between the two is clear.  To use zh-TW and zh-CN (in parallel), the country code `-tw`/`-cn` is required in every places.  If anyone can point out the difference between :hong_kong: zh-HK and :taiwan: zh-TW (say :bus: :hong_kong: *巴士* vs :taiwan: *公交車*) in our theme, I'll be happy to welcome `i18n/zh-HK.toml` into our theme.
* * *

From [Unicode's plural rules](http://www.unicode.org/cldr/charts/28/supplemental/language_plural_rules.html) found in the linked Hugo issue, we see that there're two missing lines in `i18n/pl.toml` breaking the plural behavior.

```diff
diff --git a/i18n/pl.toml b/i18n/pl.toml
index 91f21d0..ba18ca4 100644
--- a/i18n/pl.toml
+++ b/i18n/pl.toml
@@ -2,6 +2,8 @@
   other = "2-1-2006"
 [postreadingtime]
   one = "1 minuta czytania"
+  few = "{{ .Count }} minuty czytania"
+  many = "{{ .Count }} minuty czytania"
   other = "{{ .Count }} minuty czytania"
 [recent_posts]
   other = "Ostatnie posty"
@@ -27,6 +29,8 @@
   other = "dowiedz się więcej"
 [repliestext]
   one = "Odpowiada zazwyczaj w ciagu 1 godzina."
+  few = "Odpowiada zazwyczaj w ciagu {{ .Count }} godziny."
+  many = "Odpowiada zazwyczaj w ciagu {{ .Count }} godziny."
   other = "Odpowiada zazwyczaj w ciagu {{ .Count }} godziny."
 [submit]
   other = "Prześlij"
```

Adding the missing cases `few` and `many` back gives the desired result.

:information_source: Cross-posted on [Hugo Discourse](https://discourse.gohugo.io/t/plural-doesnt-work-for-brazillian-portuguese-and-polish/20329).

![Screenshot_2019-08-21 Kategorie](https://user-images.githubusercontent.com/5748535/63437395-c1c38e80-c42a-11e9-8b1b-fda6ee427b78.png)

* * *

Now, only :brazil: Portuguese is left.

VincentTam commented:
> I do think that we should go ahead and switch to [pt-br since it is currently incorrect](https://discourse.gohugo.io/t/plural-doesnt-work-for-brazillian-portuguese-and-polish/20329/3).

Unless anyone can show the difference between :brazil: Brazilian and :portugal: European Portuguese in our theme, using "pt" makes more sense.
